build/commands/lib/config.js

```diff
@@ -134,7 +134,7 @@ const Config = function () {
   this.rewardsGrantDevEndpoint = getNPMConfig(['rewards_grant_dev_endpoint']) || ''
   this.rewardsGrantStagingEndpoint = getNPMConfig(['rewards_grant_staging_endpoint']) || ''
   this.rewardsGrantProdEndpoint = getNPMConfig(['rewards_grant_prod_endpoint']) || ''
-  this.chromePgoPhase = 0
+  this.chromePgoPhase = getNPMConfig(['chrome_pgo_phase']) ? parseInt(getNPMConfig(['chrome_pgo_phase'])) : 0
```

Comment:
we still shouldn't be specifying a default here. If `chrome_go_phase` is not in npm config then we shouldn't pass any value for it at all

Reply:
Removed.